frontend/lib/src/components/elements/Expander/styled-components.ts

```diff
@@ -16,31 +16,81 @@
 
 import styled from "@emotion/styled"
 
-export const StyledIconContainer = styled.div(({ theme }) => ({
-  display: "flex",
-  gap: theme.spacing.sm,
-  alignItems: "center",
-}))
-
 export interface StyledExpandableContainerProps {
   empty: boolean
   disabled: boolean
 }
 
-export const StyledExpandableContainer =
-  styled.div<StyledExpandableContainerProps>(({ theme, empty, disabled }) => {
-    if (empty) {
-      // Don't apply hover styling if empty:
-      return {
-        ".streamlit-expanderHeader:hover": {
-          color: disabled ? theme.colors.disabled : theme.colors.bodyText,
-        },
-      }
-    }
-
-    return {
-      ".streamlit-expanderHeader:hover svg": {
-        fill: theme.colors.primary,
-      },
-    }
+export const StyledExpandableContainer = styled.div({})
+interface StyledDetailsProps {
+  isStale: boolean
+}
+
+export const BORDER_SIZE = 1 // px
+export const StyledDetails = styled.details<StyledDetailsProps>(
+  ({ isStale, theme }) => ({
+    marginBottom: 0,
+    marginTop: 0,
+    width: "100%",
+    borderStyle: "solid",
+    borderWidth: `${BORDER_SIZE}px`,
+    borderColor: theme.colors.fadedText10,
+    borderRadius: theme.radii.lg,
+    ...(isStale
+      ? {
+          borderColor: theme.colors.fadedText05,
+          transition: "border 1s ease-in 0.5s",
+        }
+      : {}),
   })
+)
+
+export const StyledSummaryHeading = styled.span(({ theme }) => ({
+  display: "flex",
+  gap: theme.spacing.sm,
+  alignItems: "center",
+  flexGrow: 1,
+}))
+
+export const StyledSummary = styled.summary(({ theme }) => ({
```

Comment:
Nit: Currently, the expander label changes to the primary color on hover, but does not remain red when not focused. In this version, the summary remains the primary color